components/tools/OmeroPy/test/integration/test_tickets3000.py

```diff
@@ -68,11 +68,6 @@ def test2396(self):
         l_ia.setChild(fa)
         self.update.saveObject(l_ia)
 
-    # This test is no longer valid as it should not be possible to remove
```

Comment:
I can't easily tell what the test is actually doing, but the admin service's `removeGroups` should throw a `ValidationException` if the operation would leave the user a member of no groups (and the `user` group doesn't count!).